examples/README.md

```diff
@@ -13,6 +13,14 @@ Machine Learning Challenge Winning Solutions
 | Place         | Competition   | Solution  | Date |
 | ------------- |:------------- | --------- | -----|
 |1st      | [M5 Forecasting - Accuracy](https://www.kaggle.com/c/m5-forecasting-accuracy) | [link](https://www.kaggle.com/c/m5-forecasting-accuracy/discussion/163684) | 2020.6 |
+|3rd      | [M5 Forecasting - Uncertainty](https://www.kaggle.com/c/m5-forecasting-uncertainty) | [link](https://www.kaggle.com/c/m5-forecasting-uncertainty/discussion/166875) | 2020.7 |
+|3rd      | [ALASKA2 Image Steganalysis](https://www.kaggle.com/c/alaska2-image-steganalysis) | [link](https://www.kaggle.com/c/alaska2-image-steganalysis/discussion/168870) | 2020.7 |
```

Comment:
can you please move the `2020.7` entries  above  the existing M5 forecasting entry? The list should be in chronological order, with the newest results on the top

![image](https://user-images.githubusercontent.com/7608904/92339562-6b001580-f07c-11ea-8139-9c2c7ab40bc9.png)